src/components/Identity/Identity.avatar.vue

```diff
@@ -1,47 +1,34 @@
+        <svg viewBox="0 0 152 152">
+          <path
+            v-if="size > 56"
+            d="M30.5 15.5c.9-.7 1.7-.9 2.7-.8.1 0 .2.1.2.2l-.1 1.2c0 .1-.1.2-.2.2-.5 0-1.1.1-1.5.5-1 .8-1.1 2.1-.3 3.1.7 1 2.1 1.2 3.1.5.4-.3.7-.8.8-1.3 0-.1.1-.2.2-.2l1.2.2c.1 0 .2.1.2.2-.2 1-.7 1.9-1.5 2.5-1.7 1.3-4.2 1-5.5-.7-1.3-1.8-1-4.3.7-5.6m6.7-4.2c-.1-.1 0-.2.1-.3l1.1-.7c.1-.1.2 0 .3.1l3.6 6.2c.1.1 0 .2-.1.3l-1.1.7c-.1.1-.2 0-.3-.1.1.1-3.6-6.2-3.6-6.2zm6-3.1c-.1-.1-.1-.3.1-.3l1.3-.6h.2l3.2 3h.1l.1-4.4c0-.1.1-.1.1-.2l1.3-.6c.2-.1.3 0 .3.2l-.3 8.1c0 .1-.1.2-.1.2h-.3l-6-5.4zm11.3-4.4c0-.1 0-.2.1-.3l1.3-.4c.1 0 .2 0 .3.1l2 7c0 .1 0 .2-.1.3l-1.3.4c-.1 0-.2 0-.3-.1 0-.1-2-7-2-7zm10.7-2.7c1.1-.2 1.9.1 2.8.6.1.1.1.2.1.3l-.7 1c-.1.1-.2.1-.3 0-.5-.3-1-.4-1.5-.3-1.2.2-2 1.3-1.8 2.5s1.2 2.1 2.4 1.9c.5-.1 1-.3 1.4-.7.2-.1.2-.1.4 0l1 .8c.1.1.1.2 0 .3-.6.8-1.5 1.3-2.5 1.4-2.1.3-4.1-1.1-4.5-3.3-.5-2.1 1-4.2 3.2-4.5m9.6 4.2c0-.7.5-1.2 1.1-1.2.7 0 1.2.5 1.2 1.2 0 .6-.5 1.1-1.2 1.1-.6.1-1.1-.4-1.1-1.1m9.7-4.2c0-.1.1-.2.2-.2l1.3.2c.1 0 .2.1.2.2l-.8 5.8 2.6.4c.1 0 .2.1.2.2L88 8.9c0 .1-.1.2-.2.2l-4.1-.6c-.1 0-.2-.1-.2-.2l1-7.2zm9.4 1.7c0-.1.1-.2.3-.2l1.3.3c.1 0 .2.1.1.3l-1.8 7c0 .1-.1.2-.3.1l-1.3-.3c-.1 0-.2-.1-.2-.3.1.1 1.9-6.9 1.9-6.9zm7.2 2.2c.1-.1.2-.2.3-.1l1.2.5c.1.1.2.2.1.3l-1.1 2.7 3.4-1.9h.2l1.3.5c.2.1.2.3 0 .4l-3.6 2 1.3 4.4c0 .1-.1.3-.2.3h-.2l-1.4-.6c-.1 0-.1-.1-.2-.1l-1-4.1-1.2 3c-.1.1-.2.2-.3.1l-1.2-.5c-.1-.1-.2-.2-.1-.3.2 0 2.8-6.6 2.7-6.6zm10.5 4.6c.1-.1.2-.1.3-.1l3.9 2.3c.1.1.1.2.1.3l-.6 1c-.1.1-.2.1-.3.1l-2.6-1.5-.7 1.2 2.2 1.2c.1.1.1.2.1.3l-.6 1c-.1.1-.2.1-.3.1l-2.2-1.2-.7 1.3 2.6 1.5c.1.1.1.2.1.3l-.6 1c-.1.1-.2.1-.3.1l-3.9-2.3c-.1-.1-.1-.2-.1-.3l3.6-6.3zm9.6 10c.3.3.9.2 1.2-.2s.2-.9-.1-1.2l-1.3-1-1.1 1.3 1.3 1.1zm-.7-4.4c.1-.1.2-.1.3 0l2.6 2c1 .8 1.2 2.3.4 3.3-.6.8-1.6 1-2.6.7l-.5 3.1c0 .2-.2.2-.3.1l-1.1-.9c-.1 0-.1-.1-.1-.2l.7-3.2-.6-.5-1.7 2.2c-.1.1-.2.1-.3 0l-1-.8c-.1-.1-.1-.2 0-.3-.3.2 4.2-5.5 4.2-5.5zM15.4 38.3c-.8 0-1.5-.7-1.5-1.5 0-.3.1-.6.2-.8 2.4-3.7 5.1-7.1 8.1-10.3.6-.6 1.5-.6 2.1-.1.6.6.6 1.5.1 2.1-2.9 3.1-5.5 6.4-7.7 9.9-.4.4-.8.7-1.3.7M5.8 60.2c-.8 0-1.5-.7-1.5-1.5v-.4c1.2-5 3-9.8 5.2-14.4.4-.7 1.3-1 2-.7.7.4 1 1.2.7 2-2.1 4.4-3.8 9-5 13.8-.1.8-.7 1.2-1.4 1.2m18.8 68c-.4 0-.8-.2-1.1-.4-13.7-14-21.3-32.4-21.3-51.7 0-.8.7-1.5 1.5-1.5s1.5.7 1.5 1.5c0 18.6 7.3 36.2 20.5 49.6.6.6.6 1.5 0 2.1-.3.3-.7.4-1.1.4m51.3 21.5c-15.4 0-30.2-4.8-42.8-13.9-.7-.5-.9-1.4-.4-2.1s1.4-.9 2.1-.4h.1C47 142 61.2 146.6 76 146.6c18.6 0 36.2-7.3 49.6-20.5.6-.6 1.5-.6 2.1 0s.6 1.5 0 2.1c-14 14-32.4 21.5-51.8 21.5m-46-16.8c-.3 0-.7-.1-1-.3-.5-.4-1-.8-1.4-1.2-.6-.6-.6-1.5-.1-2.1s1.4-.6 2.1-.1c.5.4.9.8 1.4 1.2.6.5.7 1.5.2 2.1-.3.2-.7.4-1.2.4m106.7-16.2c-.8 0-1.5-.7-1.5-1.5 0-.3.1-.6.2-.8 7.5-11.5 11.4-24.7 11.4-38.3 0-10-2.1-19.9-6.2-29-.3-.8 0-1.6.8-2 .8-.3 1.6 0 2 .8 4.3 9.5 6.5 19.8 6.5 30.3 0 14.2-4.1 28-11.9 40-.4.2-.8.5-1.3.5m-.7-79.5c-.5 0-1-.2-1.2-.7-1.9-2.9-4.1-5.6-6.4-8.2-.5-.6-.5-1.6.2-2.1.6-.5 1.5-.5 2.1.1 2.4 2.7 4.7 5.5 6.7 8.5.5.7.3 1.6-.4 2.1-.4.2-.7.3-1 .3"
```

Comment:
Reference for the halo in Medium
![image](https://user-images.githubusercontent.com/17264716/97860755-a8f87d80-1d3d-11eb-805f-fd3bc8d76e83.png)

I prefer using inline SVG for controlling fill by `currentColor`, since <img> can't do that, I think I am going to split this part to another component, see if that is acceptable

Reply:
I think doing sth like https://github.com/likecoin/liker-land/pull/314/commits/737cc71ce7c04bbf546f098ea19e277f1c9f87ae will make lighthouse happy, not sure if it does make it faster for ux though